pkg/domain/entities/images.go

```diff
@@ -251,6 +251,7 @@ type ImageSearchReport struct {
 type ImageListOptions struct {
 	All    bool     `json:"all" schema:"all"`
 	Filter []string `json:"Filter,omitempty"`
+	Size   bool     `json:"size" schema:"size"`
```

Comment:
Not your change but why does this have json and shema tags? I don't see this marshalled anywhere and it is also not decoded by the API handler. Also the field names are inconsistent, `Filter` is uppercase for some reason.

Can we just remove the tags?

Reply:
I wondered about that as well but decided for consistency. Worth investigating